### ISSUE TITLE
Q: Why Webdrone.create with a timeout: parameter (required)?

### ISSUE THREAD

**Issue by jrovegno** [a0/a0-webdrone-ruby#8]
Why timeout: parameter isn't optional?
Default timeout between 3 and 5 in most cases is not a problem.

**Comment by aldrinmartoq** [comment]
It forces people to think about timeout issues. In "real" test suites (+50 tests) we create Ruby classes with common operations (login, create transaction, etc) and handle that in config files.

But, I'm open to any suggestions for improvements ;-)

**Edit**: and maybe 5 seconds is right for you, but not for everyone needs.

**Comment by jrovegno** [comment]
If you are playing around, this is beautiful and straightforward 
```
a0 = Webdrone.create
a0.open.url         'http://www.google.com/'

```
I understand your point: slow connection, to many operation, etc. there are external variables that makes you tweak this parameter in some context, but for me is weird to force the user think about this at the beginning.

**Comment by aldrinmartoq** [comment]
I changed my mind and put a default timeout of 30 seconds. Fixed in version 1.6.0.
